addons/scratchr2/profile.css

```diff
@@ -98,25 +99,25 @@
 .activity-stream .actor a {
   color: #575e75 !important;
 }
-#content > .box:not(#profile-data):not(.slider-carousel-container):not(#page-404) {
+#content > .box:not(#profile-data):not(.slider-carousel-container):not(.sa-stats):not(#page-404) {
```

Comment:
Can't we add `.slide-carousel-container` to the `.sa-stats` element to avoid adding this?

Reply:
Possibly, but Scratch has some styles specific to slider carousels and I don't want them to affect this element.

Reply:
I think reverting each of those styles is OK, I've seen us do that previously. Or if you want to go hardcore mode add `all: initial;` to the element :P

Reply:
Apparently I was wrong, adding the class didn't change anything.